west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 0a3f2f0397a86425cc7d12fa3a0c0ab8020d80e1
+      revision: a2386efbce1866613f7c714b958a159364dc5f37
```

Comment:
This west update could be done as a separate PR and then do this PR. Recommendation is to update west.yml and code only and only if Zephyr has a interface changes (i.e. something SOF uses is changed). In this case, it seems this is a new capabiltiy, so Zephyr could be update dfirst, and then this PR could be done on top once Zephyr baseline has "tdm_slot_group".